skipper.env

```diff
@@ -0,0 +1,73 @@
+PULL_SECRET
```

Comment:
Perfect.
While it may change the behavior I think it's better than the current behavior of setting empty env vars in case it's not set in the user env.

Reply:
+1